SXR/SDK/sxrsdk/src/main/jni/objects/components/render_target.cpp

```diff
@@ -47,8 +47,12 @@ RenderTarget::RenderTarget(RenderTexture* tex, bool is_multiview)
     }
 }
 void RenderTarget::beginRendering(Renderer *renderer) {
-    if(mRenderTexture == nullptr)
+    if(mRenderTexture == nullptr) {
+        //if there is no render texture as in case of monoscopic backend, we are using default fbo
+        glBindFramebuffer(GL_FRAMEBUFFER, 0);
```

Comment:
That would be the part that breaks Daydream.

Reply:
Looking into it. I think I have a fix.